global.json

```diff
@@ -1,6 +1,11 @@
 {
+  "sdk": {
+    "version": "9.0.100-alpha.1.23511.2",
+    "allowPrerelease": true,
+    "rollForward": "major"
```

Comment:
Why major version rollforward?

Reply:
We have the same setting in format, runtime and other repositories. It allows a newer major versioned SDK to be used to build this repository.